### PR TITLE
convert PtElementInfo to hooks

### LINKED ISSUE #141: Convert <PtElementInfo /> to hooks

### PR DESCRIPTION
**Pull request by ziedtouibi**

### CODE REVIEW

**Comment by ziedtouibi** [comment]
the error in the CI tests is already refactored in #122

File: src/components/pt-element/PtElementInfo.tsx [hunk 1]
```diff
@@ -1,41 +1,36 @@
...
-import classNames from "classnames";
 import * as React from "react";
+import classNames from "classnames";
+
 import { IElement } from "../../Element";
 import { getElementLocales } from "../../ElementManager";
 import Button from "../shared/button/Button";
 import "./PtElement.scss";
-
 export interface IPtElementInfoProps {
```
Comment (mpeyper) [inline src/components/pt-element/PtElementInfo.tsx]: Consider removing the `I` prefix.

File: src/components/pt-element/PtElementInfo.tsx [hunk 2]
```diff
@@ -1,41 +1,36 @@
...
   onClick?: (element: IElement) => void;
 }
 
-@autobind
-class PtElementInfo extends React.Component<IPtElementInfoProps> {
-  public render() {
-    const { element } = this.props;
-    const elementLocales = getElementLocales(element);
+function PtElementInfo({ element, onClick }: IPtElementInfoProps) {
+  const elementLocales = getElementLocales(element);
```
Comment (mpeyper) [inline src/components/pt-element/PtElementInfo.tsx:15]: Not sure how expensive this call is, but consider `useMemo` if worried

**Review by mpeyper — APPROVED** [review]
Minor comments, nothing major. Can be ignored.

File: src/components/pt-element/PtElementInfo.tsx (hunk 2 above)
Comment (mpeyper) [reply src/components/pt-element/PtElementInfo.tsx:15]: I've taken a look and I'm not worried.

Comment (ziedtouibi) [reply src/components/pt-element/PtElementInfo.tsx:15]: @mpeyper  so should I update it with `useMemo` or there is no need to?

File: src/components/pt-element/PtElementInfo.tsx [hunk 3]
```diff
@@ -1,41 +1,37 @@
...
-  private onClick() {
-    if (this.props.onClick) {
-      this.props.onClick(this.props.element);
+function PtElementInfo({ element, onClick }: PtElementInfoProps) {
+  const elementLocales = getElementLocales(element);
+
+  const onElementButtonClick = () => {
+    if (onClick) {
+      onClick(element);
     }
```
Comment (mpeyper) [inline src/components/pt-element/PtElementInfo.tsx:20]: just an FYI, you can simplify this in TS with optional chaining:

```suggestion
    onClick?.(element);
```

**Review by mpeyper — APPROVED** [review]

File: src/components/pt-element/PtElementInfo.tsx (hunk 2 above)
Comment (mpeyper) [reply src/components/pt-element/PtElementInfo.tsx:15]: No need to

File: src/components/pt-element/PtElementInfo.tsx (hunk 3 above)
Comment (ziedtouibi) [reply src/components/pt-element/PtElementInfo.tsx:20]: cool I didn't know that we can do that with functions 🚀

**Pull request merged**